x-pack/metricbeat/module/aws/aws.go

```diff
@@ -62,25 +62,19 @@ func NewMetricSet(base mb.BaseMetricSet) (*MetricSet, error) {
 	}
 
 	awsConfig := defaults.Config()
-	awsCreds := awssdk.Credentials{
+	awsCredentials := awssdk.Credentials{
 		AccessKeyID:     config.AccessKeyID,
 		SecretAccessKey: config.SecretAccessKey,
 	}
 	if config.SessionToken != "" {
-		awsCreds.SessionToken = config.SessionToken
+		awsCredentials.SessionToken = config.SessionToken
 	}
 
 	awsConfig.Credentials = awssdk.StaticCredentialsProvider{
-		Value: awsCreds,
+		Value: awsCredentials,
 	}
 
-	awsConfig.Region = config.DefaultRegion
-
-	svcEC2 := ec2.New(awsConfig)
-	regionsList, err := getRegions(svcEC2)
-	if err != nil {
-		return nil, err
-	}
+	awsConfig.Region = "us-east-1"
```

Comment:
Is there any chance the user may need to configure this? I wonder if it would be better to use `metricSet.RegionsList[0]`